dockstore-yaml-proposals/jbrowse/.dockstore.yml

```diff
@@ -0,0 +1,65 @@
+# .dockstore.yml for JBrowse
+version: 1.1
+service:
+  type: docker-compose
+  name: JBrowse
+  author:
+  description: o
+    JBrowse is a fast, scalable genome browser built completely with JavaScript and HTML5.
+    It can run on your desktop, or be embedded in your website.
+  # These are files the Dockstore will index. They will be directly downloadable from Dockstore
+  files:
+     - docker-compose.yml
+     - healthcheck.sh
+     - port.sh
+     - prepare_refdata.sh
+     - stand_up.sh
+     - stop.sh
+     - GRCh37.json
+
+  scripts:
+    # The scripts preprovision, prestart, start, poststart, and postprovision will be executed in order.
+    # The only script required to have a value is start.
+
+    # Before any provisioning
+    preprovision:
```

Comment:
is there any point to including the key at all if the value is blank?

Reply:
I left it in there because this is intending to both be an actual .dockstore.yml file and also one that shows all the possible fields by example for people creating their own services.... Maybe I should just separate out the schema instead of combining the schema with the usage...

Reply:
That or there can be one example service where you use all the stages and other services where you don't and you show the simplified version

Reply:
Agree that a "full fledged example" with all the steps (and no blank keys) and a "barebones" with minimal keys one would be good enough for now.

Reply:
This is done in my latest push.